app/helpers/layout_helper.rb

```diff
@@ -16,6 +16,24 @@ def search_bar(*elements)
     content_for(:search_bar) { elements.join(" ").html_safe }
   end
 
+  def get_resource
+    instance_variable_get("@#{controller_name.singularize}")
+  end
+
+  def mount_breadcrumbs(obj, &block)
+    label = obj.try(:to_label) || obj.try(:to_s) unless obj.new_record?
+    default_menu = [{caption: _(controller_name.humanize), url: try("#{controller_name}_path")},
+                    {caption: [_(action_name.humanize), label].join(' '), url: '#' }].compact unless block_given?
```

Comment:
I'm not sure about showing the action in breadcrumbs. It's not consistent with katello and I don't think it adds much value. It also makes the breadcrumbs to change when you switch for example from "user edit" `Users >> Edit John` to "add breadcrumbs" `Users >> John >> Add SSH Key`.
I'd prefer displaying just the record's name without the action prefix.

Reply:
I removed the action referring  from the breadcrumbs, now its combined with the title itself.